admin/app/components/solidus_admin/products/index/component.html.erb

```diff
@@ -8,7 +8,11 @@
   <%= render component("ui/table").new(
     page: @page,
     columns: [
-      { header: "", data: -> { image_column(_1) }, class_name: "w-10" },
+      {
+        class_name: "w-[72px]",
```

Comment:
Can you help me understand why we can't use `px-4` here? I understand that having the content `flex items-center` now it will be centered, but isn't the same to use padding if we know the image size upfront?

Reply:
The table layout was changed from `auto` to `fixed`, that means that the browser will try to have equally sized columns. This creates less jumping when the header is hidden but also requires forcing the width for some columns.

Reply:
👍 but isn't that auto-centering without jumping working anyway by using the padding? Just to be clear, I know your solution works, I'm just trying to stay away from custom values in px as much as possible and stick to what's defined in the figma file, but if there are good reasons not to do that, I'm totally ok with this version.

Reply:
This is solved. I didn't know that the fixed table layout would set the width of the column fixed only if specified directly on the column (or first row). This sounds like the only way to make it work without jumps. 👍